### PR TITLE
Fix jwtgen example

### PR DESCRIPTION
**Pull request by qpfiffer**
Ran into this implementing some jwt stuff, thought I'd send it back upstream.

### CODE REVIEW

File: examples/main-auth.c [hunk 1]
```diff
@@ -145,7 +145,7 @@ int main(int argc, char *argv[])
 	fprintf(stderr, "jwt decoded successfully!\n");
 
 	/* Validate jwt */
-	if (jwt_validate(jwt, jwt_valid) <= 0) {
+	if (jwt_validate(jwt, jwt_valid) > 0) {
```
Comment (benmcollins) [inline examples/main-auth.c]: This should actually be `!= 0` The documentation in `jwt.h` says `0 on success`

Comment (qpfiffer) [reply examples/main-auth.c]: :+1: Resolved.

**Comment by benmcollins** [comment]
I'd appreciate if you could just squash this to one correct commit so I'm not merging in a commit that is known to have a bug.

Thanks

**Comment by qpfiffer** [comment]
And squashed.

**Comment by benmcollins** [comment]
Thanks!

**Pull request merged**